omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,155 +51,196 @@ running::
+   Homebrew installs Python in the following location::
+
+    '/usr/local/opt/python/libexec/bin'
+
+   Follow the instructions from the brew Python install and set the Homebrew version of Python 
+   to be used rather than the Python shipped with OS X. Add the following line to your :file:`.bash_profile`::
+
+    export PATH="/usr/local/opt/python/libexec/bin:$PATH"
+
+.. note:: 
+   To keep things a little cleaner, add the following enviornment variable to your :file:`.bash_profile`::
+
+    # Environment variable pointing to Homebrew Python location
+    export PYTHON_BREW=/usr/local/opt/python/libexec/bin
+
+   and append it to the :envvar:`PATH`::
+
+    export PATH="$PYTHON_BREW:$PATH"
+
+5. Check that Python is working and is version 2.7.x::
 
     $ which python
-    /usr/local/bin/python
+    /usr/local/opt/python/libexec/bin/python
+
     $ python --version
-    Python 2.7.9
+    Python 2.7.13
 
-The installation of OMERO via Homebrew depends upon two alternate
-repositories containing extra formulae:
-https://github.com/Homebrew/homebrew-science for the HDF5 formula and
-https://github.com/ome/homebrew-alt for all the OME-provided formulae
-and older versions of Ice. To add these, run::
+6. For developing with OMERO, or Python in general, we would recommend you use Virtualenv.
+   Virtualenv allows us to develop Python applications without having to 
+   worry about clashing third-party packages for different Python projects.
 
-    $ brew tap homebrew/science
-    $ brew tap ome/alt
+   Use pip to get `Virtualenv <https://virtualenv.pypa.io/en/stable/>`__::
 
-.. note::
+    $ pip install virtualenv
+
+   With Virtualenv installed, create a virtual enviornment::
+
+    $ virtualenv ~/Virtual/omero
+
+   This will create a folder to hold Python libraries in the the directory :file:`~Virtual/omero/lib`
+
+.. note:: You can activate the Virtualenv enviornment that we created using::
+
+    $ source ~/Virtual/omero/bin/activate
+
+   This will swtich to using Pip and Python in the Virtualenv directory 
+   :file:`~/Virtual/omero/bin` and any Pip libraries you install, whilst the Virtualenv
+   is activated, will be installed to :file:`source ~/Virtual/omero/lib`.
+
+   .. note:: (OPTIONAL) You can add an `alias` to your :file:`.bash_profile` to make this step easier::
+
+        alias startVmOmero="source ~/Virtual/omero/bin/activate"
 
-    The Homebrew formulae used below provide Python bindings. As
-    described in `Homebrew and Python`_, you should **not** be in an
-    active virtual environment when you ``brew install`` them.
+      Reload :file:`.bash_profile` in OS X::
 
-See the :download:`step01_deps.sh <walkthrough/osx/step01_deps.sh>` script for
-the steps described above.
+        $ source ~/.bash_profile
+
+      Now you can activate the Virtualenv enviornment using::
+
+        $ startVmOmero
+
+.. Note::
+   See the :download:`step01_deps.sh <walkthrough/osx/step01_deps.sh>` script for the steps described above.
```

Comment:
I do not see the steps 

``brew update`` 
and 
``brew doctor``
which are included in the script in this walkthrough